### DISCUSSION TITLE
New version announcement

### DISCUSSION THREAD

**Discussion by marcoshaw** [crc-org/crc#3390] [General]
Last I checked, the Windows version was 4.11.3 and Linux was 4.11.7.

Am I missing something obvious that can check when a new version is available?

**Comment by cfergeau** [comment]
New releases are announced to https://github.com/code-ready/crc/releases and are uploaded to https://mirror.openshift.com/pub/openshift-v4/clients/crc/latest Both should be using 4.11.7

How did you check what version is the latest?

**Reply by marcoshaw** [reply]
I must have missed something obvious.  My "version check" is simply based on recent installs of each product.  I'll retry both soon.  For whatever reason, it looks like Windows was getting 4.11.3 a few weeks ago, no matter how many times I tried to reset my install, while Linux was 4.11.7.

I must be doing something wrong when I delete.

**Reply by cfergeau** [reply]
Once you downloaded a crc binary, it will always use the same OpenShift version. You need to update to a newer binary in order to have it use automatically a new OpenShift version.
However, this has been improved recently, and you can use `crc setup --bundle https://developers.redhat.com/content-gateway/file/pub/openshift-v4/clients/crc/bundles/openshift/4.11.7/crc_libvirt_4.11.7_amd64.crcbundle` and `crc start --bundle https://developers.redhat.com/content-gateway/file/pub/openshift-v4/clients/crc/bundles/openshift/4.11.7/crc_libvirt_4.11.7_amd64.crcbundle` to use a newer openshift version with an older binary.

**Reply by marcoshaw** [reply]
Thank you.  I assumed if I deleted everything and re-downloaded the package/installer, that this would ensure a new version.

I'm sorry if I didn't properly RTFM...

**Reply by cfergeau** [reply]
> if I deleted everything and re-downloaded the package/installer

If you redownload, it depends where you download from (either from `/crc/latest/...` or from `/crc/2.18.1/`). From your confusion, I assume you are using `latest` links, and yes, this should be enough to be getting the latest version.

**Reply by marcoshaw** [reply]
I always assumed/thought if I went to the Red Hat site and downloaded the crc installer from there, it would automatically do this somehow and grab the latest.

I wondered if the Red Hat installer they provide has a hard coded version.

I guess re-downloading the installer from https://console.redhat.com/openshift/create/local does not guarantee the latest CRC version.

Thanks for the feedback.

**Reply by cfergeau** [reply]
> I guess re-downloading the installer from https://console.redhat.com/openshift/create/local does not guarantee the latest CRC version.

Downloading from this location should be fine as it links to https://developers.redhat.com/content-gateway/rest/mirror/pub/openshift-v4/clients/crc/latest/crc-linux-amd64.tar.xz and such
